VideoLocker/res/drawable/row_discussion_first_comment_background.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <solid android:color="@color/edx_grayscale_neutral_white" />
+    <stroke
+        android:width="@dimen/edx_hairline"
+        android:color="@color/edx_grayscale_neutral_light" />
+    <corners android:radius="@dimen/edx_box_radius"
```

Comment:
> Due to some Android quirks, you need to define it or else 0dp radius won't work.

I can't reproduce this issue in Lollipop (5.0) or Jelly Bean (4.1.2). Is it specific to some other API level?